client/me/reauth-required/index.jsx

```diff
@@ -254,17 +262,33 @@ const ReauthRequired = createReactClass( {
 				</form>
 			</Dialog>
 		);
-	},
+	}
 
-	handleChange( e ) {
+	handleChange = e => {
```

Comment:
you can do 
```js
handleChange = ( { currentTarget } ) => {}
```
or
```js
handleChange = ( { currentTarget: { name, value } } ) => {}
```
if you want to push syntax optimization even further. Completely optional.